github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -305,6 +305,44 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
+  - name: pull-containerized-data-importer-e2e-ceph-wffc
+    skip_branches:
```

Comment:
Could we shorten this `skip_branches:` with some like we have for the kubevirt jobs? 

https://github.com/kubevirt/project-infra/blob/d346545770114ef120b3d89be58c339b33401038/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L23C4-L23C4

Reply:
I believe we can't - we don't want to skip all of them, just this specific set

Reply:
Ok thanks for clearing that up for me.